### PR TITLE
Accessibility bug fixes

### PR DESCRIPTION
**Pull request by iremgokce**
- **Color Picker** component guid option for unique ids
- **Hierarchy Component** aria roles for li tag and filter button

### CODE REVIEW

File: src/UXClient/Components/ColorPicker/ColorPicker.ts [hunk 1]
```diff
@@ -28,8 +40,8 @@ class ColorPicker extends Component{
         let colorPickerButton = this.colorPickerElem.append('button').classed("tsi-colorPickerButton", true)
                     .attr("title", this.getString('Select color'))
```
Comment (ccrowley96) [inline src/UXClient/Components/ColorPicker/ColorPicker.ts:41]: optional nit: move `.attr` chained calls to only one tab indent

File: src/UXClient/Components/ColorPicker/ColorPicker.ts [hunk 2]
```diff
@@ -44,11 +56,11 @@ class ColorPicker extends Component{
         }
 
         colorPickerButton.append('span').classed("tsi-selectedColorValue", true).classed("hidden", this.chartOptions.isColorValueHidden)
-                            .attr("id", "tsi-selectedColorValue")
```
Comment (ccrowley96) [inline src/UXClient/Components/ColorPicker/ColorPicker.ts:47]: optional nit: move `.attr` chained calls to only one tab indent

**Review by ccrowley96 — APPROVED** [review]
Code changes look good!  Not sure how to verify the a11y stuff.  But looks good to me.

File: src/UXClient/Components/ColorPicker/ColorPicker.ts [hunk 3]
```diff
@@ -4,16 +4,28 @@ import Utils from '../../Utils';
...
-    private isColorGridVisible;
+    private colorPickerElem: any;
+    private selectedColor: string | null;
+    private isColorGridVisible: boolean;
+    private guid: string;
 
-    constructor(renderTarget: Element){
+    constructor(renderTarget: Element, guid: string = Utils.guid()){
         super(renderTarget);
+        this.guid = guid;
```
Comment (darsney) [inline src/UXClient/Components/ColorPicker/ColorPicker.ts:25]: probably call this componentId ??

**Pull request merged**